src/scripts/main.js

```diff
@@ -358,3 +358,29 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+
+const tableElement = document.querySelector('.dashboard'); // отримуємо доступ до таблиці за класом
+
+people.forEach((persone) => { // Проходиом по масиву людей
```

Comment:
The variable name 'persone' is likely a typo. Consider renaming it to 'person' for consistency and clarity.